CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.14.2
+* Bug - Added checks for circular task dependencies. [#796](https://github.com/aws/amazon-ecs-agent/pull/796)
+* Bug - Fixed an issue with Docker auth configuration overrides [#751](https://github.com/aws/amazon-ecs-agent/pull/751)
+* Bug - Fixed a race condition in the task clean up code path. [#737](https://github.com/aws/amazon-ecs-agent/pull/737)
+* Bug - Upgrade AWS SDK to v1.8.19 to include SDK bugfixes [#786](https://github.com/aws/amazon-ecs-agent/pull/786)
+* Bug - Fixed an issue involving concurrent map writes [#743](https://github.com/aws/amazon-ecs-agent/pull/743)
+
```

Comment:
Also, you're missing the `Enhancement` to introspection API for querying task id's with short container ids #813